src/SessionSetup/SessionSetupUtilsTest.cpp

```diff
@@ -31,19 +31,35 @@ TEST(SessionSetupUtils, CredentialsFromSshInfoWorksCorrectly) {
   EXPECT_EQ(info.user.toStdString(), credentials.user);
 }
 
-TEST(SessionSetupUtils, ConnectionTargetFromString) {
-  const QString instance = "some/i-nstanc-3/id123";
+TEST(SessionSetupUtils, ConnectionTargetFromValidInstanceID) {
```

Comment:
```suggestion
TEST(SessionSetupUtils, ConnectionTargetFromValidInstanceId) {
```
"prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC()."